src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -0,0 +1,123 @@
+from django.contrib.gis.geos import MultiPolygon, Polygon
+from django.http import QueryDict
+from typing import TypedDict
+
+import json
+
+
+# A list of coordinates representing a polygon.
+class PolygonFromUrlParams(TypedDict):
+    coordinates: list[tuple[float, float]]
+
+
+# A project area composed of multiple disjoint polygons.
+class ProjectAreaFromUrlParams(TypedDict):
+    # Project ID
+    id: int
+    # SRID
+    srid: int
```

Comment:
will we use different values here?

Reply:
in the db, geometry srid's have been 4269 while for rasters, we've been using a srs string (associated weakly with srid, 9822).

I think it's preferable to make this explicit rather than a hidden default.

Reply:
I realized 4269 is hardcoded as a default in a few places.
![image](https://user-images.githubusercontent.com/3720361/212146367-5c38308e-e317-4e40-8635-1ad9721c0116.png)

I'll do this same here (i.e. make it optional). And add a TODO for replacing it with a constant.

Reply:
done - ptal.